scripts/versioning/util/release_utils.rb

```diff
@@ -17,9 +17,9 @@ def self.is_breaking_change(commits, platform)
     # Method to determine if there are any features in the commits
     def self.is_feature(commits, platform)
       regex = if platform == ReleasePlatform[:IOS]
-                 /^(?i)(feat)(\[[^\]]*ios[^\]]*\])?(\(.*\))?:.+/
+                 /^(?i)(feat|chore)(\[[^\]]*ios[^\]]*\])?(\(.*\))?:.+/
```

Comment:
Why is it including `chore`?

Reply:
Got it. I’ll apply the fix directly in the commit name instead of adding chore, I will close this PR, and create a new version.